### DISCUSSION TITLE
ITead “SONOFF Zigbee Bridge Pro” (ZBBridge-P) new CC2652P and ESP32 based Zigbee Gateway to WiFi Bridge

### DISCUSSION THREAD

**Discussion by Hedda** [Koenkk/zigbee2mqtt#10758] [General]
FCC leak show CC2652P and ESP32 based Sonoff ZBBridge-P v1.1 Zigbee Bridge variant and wireless Zigbee sensors is coming from ITead:

https://fccid.io/2APN5ZBBRIDGEP

UPDATE! ZBBridge-P (a.k.a. "ZB Bridge-P") has now been released as "SONOFF Zigbee Bridge Pro":

https://itead.cc/product/sonoff-zigbee-bridge-pro/

According to @digiblur who broke/spotted this news, the Sonoff ZBBridge-P and the new sensors are slated to be released in March of 2022:

https://www.digiblur.com/2021/12/new-sonoff-zigbee-bridge-and-sensors.html

Based on the specs it should be compatible with Tasmota and ESPhome, at least when used as [remote adapter](https://www.zigbee2mqtt.io/advanced/remote-adapter/connect_to_a_remote_adapter.html) for Zigbee2MQTT and ZHA:

https://tasmota.github.io/docs/Zigbee/

https://github.com/thegroove/esphome-zbbridge

Internal pictures posted from FCC show using an unannounced Texas Intruments CC2652P (CC2652P1) MCU chip based “SM-031 v1.1” radio module which must be made by [CoolKit](https://github.com/CoolKit-Technologies/DevDocs/tree/master/Zigbee) (the company that makes their EFR32MG21 based “[SM-011 V1.0](https://www.coolkit.cn/product/sm-011/)” module). 

ESP32 (ESP32-D0WD-V3) has replaced the ESP8266/ESP8285, and it also looks like added an on-board clock-battery too, presumably for RTC (Real-Time Clock)?

https://fccid.io/2APN5ZBBRIDGEP/Internal-Photos/Internal-Photos-5613131

![image](https://user-images.githubusercontent.com/6320001/149344465-aa257c68-5e2b-4b3d-8f57-bb89ea48f4ba.png)

![image](https://user-images.githubusercontent.com/6320001/149344497-da6186e7-fcc2-4ecd-86d0-a3e58430c586.png)

![image](https://user-images.githubusercontent.com/6320001/149344517-dd8226e1-5476-46a7-a1d0-b6dd3ac78318.png)

![image](https://user-images.githubusercontent.com/6320001/149344874-fb384594-3edb-401e-b3a3-f68fc0c4623e.png)

![image](https://user-images.githubusercontent.com/6320001/149344589-c148f43f-299f-4b81-b117-a90932889e8f.png)

![image](https://user-images.githubusercontent.com/6320001/149344613-0be5e70f-6f7d-4b46-b756-ed72a162d8b0.png)

![image](https://user-images.githubusercontent.com/6320001/149345073-5d0ca3e3-f51c-4701-a7fe-3b2472101649.png)

![image](https://user-images.githubusercontent.com/6320001/149345172-4e92f7a2-8a2b-42cb-b7b1-6f8c249d317a.png)

FFC also leaked new Sonoff branded [Door/Window Sensor](https://fccid.io/2APN5SNZB-04P), [Motion Sensor](https://fccid.io/2APN5SNZB-03P), [Temp/Humidity Sensor](https://fccid.io/2APN5SNZB-02P), and [a Button](https://fccid.io/2APN5SNZB-01P) as wireless Zigbee devices, all with the same CCC2652P (CC2652P1/CC2652P1F) chip which +20 dBm capable power amplifier is really overkill for battery-operated sensors and should on theory use more battery power than if they would have used the compatible CC2652R or CC2652RB.

Sadly it looks like this new variant of Sonoff ZBBridge still uses Wi-Fi instead of a wired Ethernet network like the ESP32 based ZB-GW03:

https://community.home-assistant.io/t/zb-gw03-ewelink-ethernet-zigbee-gateway-now-hacked-with-tasmota-zbbridge-so-can-be-used-via-mqtt-or-as-a-remote-zigbee-adapter-with-home-assistant-zha-and-zigbee2mqtt/341223

https://github.com/arendst/Tasmota/discussions/12764

https://github.com/syssi/esphome-zb-gw03/

Ethernet is preferred by DIY users who flash Tasmota or ESPHome in order to use it as a remote Zigbee adapter with ZHA or Zigbee2MQTT because tunnelling serial communication over a serial stream server connection and the serial APIs of the Zigbee stacks are not robust/stable over WiFi as assume a direct connection, which is why both ZHA and Zigbee warns against using WiFi-based networked-attached Serial-to-IP Zigbee bridges/gateways/adapters:

https://www.home-assistant.io/integrations/zha#warning-about-wi-fi-based-zigbee-to-serial-bridgesgateways

https://www.zigbee2mqtt.io/advanced/remote-adapter/connect_to_a_remote_adapter.html

Looking at the board pictures of "ITead SONOFF Zigbee 3.0 USB Dongle Plus" from @NotEnoughTECH (Mat Zolnierczyk) my guess is it is the same engineer at CoolKit Technologies who engineered this “SM-031 v1.1” radio-module that helped ITead design the board for the SONOFF Zigbee 3.0 USB Dongle Plus adapter. Its bad start tuned out to only be issues with early firmware fixed now and not a faulty hardware design:

https://notenoughtech.com/home-automation/sonoff-zigbee-3-0-usb-dongle-plus/

https://notenoughtech.com/home-automation/is-sonoff-zigbee-dongle-plus-finally-ready-for-launch/

![image](https://user-images.githubusercontent.com/6320001/149344360-2fac5365-1ac3-4907-af33-c7b34025cc10.png)

![image](https://user-images.githubusercontent.com/6320001/149344409-82924db3-05d8-45f7-b0ef-ccde3d1e9a0b.png)

Not sure if this is intended to permanently replace the popular ITead Sonoff ZBBridge

https://community.home-assistant.io/t/sonoff-zbbridge-sonoff-zigbee-bridge-from-itead/187346

As I understand ITead have not abandoned Silicon Labs yet but it is just that Silabs chip stocks have suffered worse than most other companies during the still currently ongoing worldwide chip shortage.

PS: I assume it will use the same community firmware from Koenkk for CC2652P as they use for “Sonoff Zigbee 3.0 USB Dongle Plus”:

https://github.com/Koenkk/zigbee2mqtt/discussions/8840

**Comment by digiblur** [comment]
I can't say it is a leak being a public website and just following procedure. Leak in my eyes would be getting a prerelease product then posting pics ahead of time about it.

Glad to hear the USB dongle is now ready for launch 👀

**Comment by notenoughtech** [comment]
I asked Sonoff directly... for samples :D Let's see if I can get my hands before the release

**Comment by kirovilya** [comment]
I don't want to sound important, but this is last year's news :)
we have already seen and discussed it in the Russian telegram channel. https://t.me/zigbeat/1636

Another mini-relay variant is expected soon https://t.me/zigbeat/1647

**Reply by Hedda** [reply]
> I don't want to sound important, but this is last year's news :)
> we have already seen and discussed it in the Russian telegram channel. https://t.me/zigbeat/1636

Yes digiblur posted about it on the 30th December of 2021 but I did not notice it earlier, so it was news to me ;)

**Comment by digiblur** [comment]
Indeed it is from 2021.

**Reply by kirovilya** [reply]
But this picture is even more secret for me :)
(picture not mine)
![image](https://user-images.githubusercontent.com/8360230/149374733-c6b1c743-ca1f-4f2c-b880-a2680ca5c515.png)

**Comment by Hedda** [comment]
FYI, ITead has apparently now started selling "**SONOFF Zigbee Bridge Pro**" which I assume are the same as the previously leaked "Sonoff ZBBridge-P" model based on CC2652P and ESP32-D0WD-V3?

https://itead.cc/product/sonoff-zigbee-bridge-pro/

![image](https://user-images.githubusercontent.com/6320001/166225949-8c9f767a-768e-4abc-8256-9af358715993.png)

Its packaging is the same as and their previous Zigbee gateway/bridge and the only listed difference mentioned on its website is that it can handle 128 instead of 32 devices, contains a “buzzer” which webpage mention that the buzzer can be used as an alarm (siren?) or as a doorbell chime (if use Zigbee wireless buttons as doorbells), and a “built-in high-precision local RTC” (Real-Time Clock with clock-battery) for gateway time synchronization of local scene schedules.

![image](https://user-images.githubusercontent.com/6320001/166228468-2a9d9eac-3e81-44c0-a342-770ce6c6e1b7.png)

Note that they are also still selling the older ZBBridge based on EFR32MG21 and ESP8266/ESP8285 which has already been hacked with Tasmota and ESPHome as a network-attached Zigbee adapter:

https://itead.cc/product/sonoff-zbbridge/

See also

https://zigbee.blakadder.com/Sonoff_ZBBridge.html

and

https://templates.blakadder.com/ewelink_ZB-GW03

Note that Zigbee2Tasmota (tasmota-zbbridge) is by the way already supported on Tube's Zigbee Gateways with similar specifications:

https://tasmota.github.io/docs/Zigbee/

https://www.tubeszb.com/product/cc2652_coordinator/39

https://www.tubeszb.com/product/cc2652_poe_coordinator/47

https://github.com/tube0013/tube_gateways

(Tube's Zigbee Gateways are based on ESP32 dev board (Wireless-Tag WT32-ETH01 or Olimex ESP32-POE-ISO) combined with either a Texas Instruments CC2652P based Zigbee module or a Silicon Labs EFR32MG21 based Zigbee module).

**Comment by notenoughtech** [comment]
Mine finally arrived and you can see early impressions here: https://notenoughtech.com/home-automation/sonoff-zigbee-bridge-pro/
If you have any questions  - feel free to ask.

**Comment by Hedda** [comment]
FYI, there are also a Tasmota and ESPHome discussions which will probably be among the first places to see news about hacking ESP32 on it: 

https://github.com/arendst/Tasmota/discussions/14419 

https://community.home-assistant.io/t/itead-sonoff-zigbee-bridge-pro-zbbridge-p-new-cc2652p-and-esp32-based-zigbee-gateway-to-wifi-bridge/378924/5
